### ISSUE TITLE
I've got error TypeError: cannot unpack non-iterable NoneType object

### ISSUE THREAD

**Issue by kamwansiri** [PfizerRD/scikit-digital-health#104]
I want to used sleep function collaborate with my accelerometry data which contain Timestamp and 3Axes which are xyz0,xyz1, and xyz2 in .csv file. So I write my code like this and I've got some error. Someone help me please, Thank you.
![image](https://github.com/PfizerRD/scikit-digital-health/assets/133620875/e6d155bb-2dd2-4783-b093-c05c88bba46a)
![image](https://github.com/PfizerRD/scikit-digital-health/assets/133620875/8e2608af-6dce-4001-ad6f-4de0a33bb1f7)
![image](https://github.com/PfizerRD/scikit-digital-health/assets/133620875/f01cdf65-909e-484c-9f8a-d3205d24ed9f)

**Comment by kamwansiri** [comment]
Oh, Thank you. Now, I can fix it.

**Comment by kamwansiri** [comment]
Excuse me, I have another question. If I want to get the Time end like Timestamp end of Sleep opportunity. Do you have any suggestion for me?
![image](https://github.com/PfizerRD/scikit-digital-health/assets/133620875/cd5257df-0707-4872-bc20-24ed76528ac3)

**Comment by LukasAdamowicz** [comment]
yes you can add duration to start timestamp to get the end, and convert it if necessary to human readable

**Comment by kamwansiri** [comment]
> yes you can add duration to start timestamp to get the end, and convert it if necessary to human readable

Thank you very much for beautiful code and beautiful suggestion for me.
